packages/istanbul-lib-report/README.md

```diff
@@ -3,4 +3,28 @@
 [![Greenkeeper badge](https://badges.greenkeeper.io/istanbuljs/istanbul-lib-report.svg)](https://greenkeeper.io/)
 [![Build Status](https://travis-ci.org/istanbuljs/istanbul-lib-report.svg?branch=master)](https://travis-ci.org/istanbuljs/istanbul-lib-report)
 
-Core reporting utilities for istanbul. Docs to follow.
+Core reporting utilities for istanbul.
+
+## Example usage
+
+```js
+const libReport = require('istanbul-lib-report');
+const reports = require('istanbul-reports');
+
+// create a context for report generation
+const context = libReport.createContext({
+  dir: 'report/output/dir',
+  defaultSummarizer: 'summarizer name e.g. nested/flat/pkg',
+  watermarks: configWatermarks,
+  coverageMap,
```

Comment:
Variables like these last 2 I would like to see defined, so that devs know what needs to be passed here or how to get the coverageMap to this point.

Reply:
I defined configWatermarks.
For coverageMap - its a ball of string.. I could include libCoverage and create a coverage map.. but that requires either passing an empty report or loading a report and merging it in.. where does it end? this is not at the top level - how to use all istanbul packages, this is how to use istanbul-reports. As the main readme states, most end users won't need this (though I am writing these docs because I partly disagree and think the main readme documentation is way to aggressive and unhelpful)

Reply:
I defined it with a comment.